docs/conf/eu/2016/news/APIdays-blurb.md

```diff
@@ -0,0 +1,3 @@
+## Write the Docs EU, September 18-20, Prague, CZ
+
+To all members of the API community who care about good docs -- and who doesn't? -- Write the Docs, a conference for everyone who cares about documentation (developers, writers, designers, architects, and more) is taking place this year in Prague, CZ, on September 18-20. There will be a wide range of talks, including several specifically about API docs, from GraphQL as an API definition language to usability studies of API docs (with data!) to the importance of naming in API design. You should attend! For more information, see the [conference website](http://www.writethedocs.org/conf/eu/2016/) or [purchase tickets](https://ti.to/writethedocs/write-the-docs-eu-2016?_=). Documentarians are a friendly bunch, and they'd love to see you there!
```

Comment:
I think the opening sentence has a few too many subordinate clauses. I'd say, maybe: 
'If you're into APIs and good documentation, we'd like to invite you to attend Write the Docs, a conference for everyone who cares about documentation. This year's EU conference is coming up soon, September 18-20, in Prague, Czech Republic.'

Then, just a couple little suggestion for the rest: 
'There will be a wide range of talks, including several specifically about API docs. We'll be hearing about GraphQL as an API definition language, usability studies of API docs (with data!), and the importance of naming in API design, to name a few. Don't miss out! For more information, see the [conference website](http://www.writethedocs.org/conf/eu/2016/) or [purchase tickets](https://ti.to/writethedocs/write-the-docs-eu-2016?_=). Documentarians are a friendly bunch, and we'd love to have you join us!